docs/pages/build-contract.mdx

```diff
@@ -19,3 +19,14 @@ You can also add the script `"build": "xsuite build"` to the `package.json` of y
 ```
 npm run build
 ```
+
```

Comment:
The first comment I have is regarding the documentation: great that you added a mention in the Build page. It would be great to also document the command in the CLI page.

Reply:
should be done. Let me know if know if you have any issues with the spelling/grammar or with the completeness.